cypress/support/commands.js

```diff
@@ -21,16 +27,100 @@ Cypress.Commands.add('initiateExceptionsLogger', () => {
     });
 });
 
-Cypress.Commands.add('visitPage', path => {
-    cy.task('getProperty', 'webapp.url').then(baseUrl => {
-        cy.visit(`${baseUrl}${path}`);
-    });
-})
-
 Cypress.Commands.add('getByLabel', (label) => {
     cy.contains('label', label)
         .invoke('attr', 'for')
         .then((id) => {
             cy.get('#' + id)
         })
 })
+
+
+Cypress.Commands.add('getByPlaceholder', (placeholder) => {
+    cy.get(`input[placeholder="${placeholder}"]`)
+})
+
+Cypress.Commands.add('login', () => {
+    cy.request({
+        method: 'POST',
+        url: `${API_BASE_URL}/session`,
+        body: {sessionLocation: DEFAULT_LOCATION_UUID},
+        headers: {
+            'Content-Type': 'application/json',
+            Authorization: `Basic ${TOKEN}`,
+        },
+    });
+});
+
+Cypress.Commands.add('createPatient', ()=>{
+    const patient = {
+        person: {
+            names: [
+                {
+                    preferred: true,
+                    givenName: 'John',
+                    middleName: '',
+                    familyName: 'Doe'
+                }
+
+            ],
+            gender: 'M',
+            birthdate: '1997-09-12T18:00:00.000Z',
+            birthdateEstimated: false,
+            attributes: [],
+            addresses: [
+                {
+                    postalCode: '0100',
+                    address2: '',
+                    address1: 'Haha',
+                    country: 'Sri Lanka',
+                    stateProvince: 'Western',
+                    cityVillage: 'Colombo 5'
+                }
+            ],
+            dead: false
+        },
+        identifiers: [
+            {
+                identifier: null,
```

Comment:
That's odd. Does the `null` identifier cause it to auto-generate? If so, please leave a comment to that effect.

Reply:
No, actually it's being replaced with the retrieved identifier from the `iden` endpoint. However, it's clear that there's a readability issue in the code. I'll add a comment.